docs/server.rst

```diff
@@ -122,25 +132,30 @@ not affect the versions that are installed for Tunneldigger.
 
 .. _virtualenv: http://pypi.python.org/pypi/virtualenv
 
-You can install all of the above simply by running on Debian::
+You can install all of the above simply by running on Debian <= 9::
 
     sudo apt-get install iproute bridge-utils libnetfilter-conntrack-dev libnfnetlink-dev libffi-dev python-dev libevent-dev ebtables python-virtualenv
 
+and for Debian 10::
+    sudo apt install iproute2 bridge-utils libnetfilter-conntrack-dev libnfnetlink-dev libffi-dev python-dev libevent-dev ebtables python-virtualenv
+
 and for Fedora you can use this command::
 
     sudo yum install iproute bridge-utils libnetfilter_conntrack python-devel libevent-devel ebtables libnl-devel python-pip python-virtualenv
 
 Installation
 ------------
 
+Check for requirements at "System Packages"
+
 If we assume that you are installing Tunneldigger under ``/srv/tunneldigger``
 (the scripts provided with Tunneldigger assume that as well), you can do::
 
     cd /srv/tunneldigger
     virtualenv -p /usr/bin/python3 env_tunneldigger
 
 .. note::
-    Tunneldigger only supports Python 3.
+    Tunneldigger in the newest version only supports Python 3.
```

Comment:
I'm not sure since when. But there are definitely tunneldigger versions e.g. v0.3.0 that run with python 2.7